### ISSUE TITLE
No validating backend XSS

### ISSUE THREAD

**Issue by BaHeK1994** [FriendsOfSymfony/FOSCKEditorBundle#192]
1) Open form
![изображение](https://user-images.githubusercontent.com/18642618/60621939-bf0fca00-9de7-11e9-9847-df25a214bbf5.png)
2) Open browser console
3) CKEDITOR.instances[textareaId].destroy(true);
4) Paste XSS
![изображение](https://user-images.githubusercontent.com/18642618/60622029-f8483a00-9de7-11e9-8faf-0568b7bf1dd6.png)
5) Submit
![изображение](https://user-images.githubusercontent.com/18642618/60622067-11e98180-9de8-11e9-9480-0f16f5c59146.png)

**Comment by TomaszGasior** [comment]
I don't see any issue related to this bundle here. Regardless of used WYSIWYG editor you MUST do validation on your own, using good tools like: https://github.com/tgalopin/html-sanitizer-bundle

**Comment by BaHeK1994** [comment]
Without an editor, the default is to escape HTML characters and there are no problems. It turns out this bandl adds only a few javascript lines and no backend check? You can, by default, add the same html-sanitizer-bundle to yourself and add backend processing to the html so that developers can safely install the bundle from the composer without unnecessary actions.

**Comment by TomaszGasior** [comment]
> Without an editor, the default is to escape HTML characters

No, it isn't. Symfony framework itself does not escape HTMl characters while saving. Escaping should be done using sanitizer while saving or reading or on template side, using twig filter. IMHO this bundle is not responsive for handling data the way you want.

You may want provide your application repository to see how to done "submit" action shown on screenshot.

**Comment by BaHeK1994** [comment]
When displaying, twig shields HTML by default.
```
{% set var = '<p>test</p>' %}
{{ var }}
```
output:
`&lt;p&gt;test&lt;/p&gt;`

Unfortunately, I can not provide access, because the private repository. To generate the form used https://github.com/sonata-project/SonataAdminBundle
![image](https://user-images.githubusercontent.com/18642618/61367772-00e34a80-a895-11e9-95e3-471384308eb4.png)

**Comment by lukepass** [comment]
Using `FOSCKEditorBundle` is as simple as this:

1. Install the bundle suggested by @TomaszGasior 
2. Edit the desired forms adding the following attribute:

```
->add('content', CKEditorType::class, [
    'config_name' => 'advanced',
    'required' => false,
    'sanitize_html' => true, // this is the important line
])
```

3. Done!
